src/test/java/test/inheritance/github949/ChildClassSample.java

```diff
@@ -0,0 +1,11 @@
+package test.inheritance.github949;
+
+import org.testng.annotations.Test;
+
+public class ChildClassSample extends ParentClassSample {
+    @Test
+    public void independent() {
```

Comment:
You can add `@Override`

Reply:
Fixed and forcibly pushed the changes so that its still a single commit.

Reply:
:+1: